workflows/requests.go

```diff
@@ -0,0 +1,34 @@
+package workflows
+
+import (
+	"go.temporal.io/sdk/activity"
+	"go.temporal.io/sdk/worker"
+	"go.temporal.io/sdk/workflow"
+)
+
+type WorkflowReference struct {
+	InstanceID string `json:"instance_id"`
+}
+
+// StartRequest is the object describing a Start Workflow request.
+type StartRequest struct {
+	Options      map[string]string `json:"workflow_options"`
+	WorkflowInfo WorkflowReference `json:"workflow_info"`
```

Comment:
Same comment I left on the other PR, but I suggest using a name such as `WorkflowReference` instead of `WorkflowInfo`. I think this will help with clarity.
```suggestion
	WorkflowReference WorkflowReference `json:"workflow_reference"`
```